fastlane/resources/values/strings.xml

```diff
@@ -4003,61 +4003,57 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
-    <string name="editor_title_beta">Beta</string>
-    <string name="dialog_title">Insert link</string>
-    <string name="dialog_edit_hint">http(s)://</string>
-    <string name="dialog_button_ok">OK</string>
-    <string name="dialog_button_cancel">Cancel</string>
-    <string name="media_insert_unimplemented">Apologies! Feature not implemented yet :(</string>
-    <string name="stop_upload_dialog_button_yes">Yes</string>
-    <string name="stop_upload_dialog_button_no">No</string>
-    <string name="retry_failed_upload_title">Couldn\'t upload file</string>
-    <string name="retry_failed_upload_yes">Retry</string>
-    <string name="retry_failed_upload_remove">Remove upload</string>
-    <string name="retry_failed_upload_retry_all">Retry all</string>
-    <string name="long_post_dlg_saving">Saving</string>
-    <string name="menu_toolbar_title">Edit %s block</string>
-    <string name="pref_key_seen_classic_editor_deprecation_notice" translatable="false">wp_pref_seen_classic_editor_deprecation_notice</string>
-    <string name="dialog_title_try_block_editor" translatable="true">Try the new Block Editor</string>
-    <string name="dialog_body_try_block_editor" translatable="true">We’ll be removing the classic editor for new posts soon, but this won’t affect editing any of your existing posts or pages. Get a head start by enabling the Block Editor now in site settings.</string>
-    <string name="dialog_dismiss_try_block_editor" translatable="true">Dismiss</string>
-    <string name="featured_image_replace_dialog_title">Replace current featured image?</string>
-    <string name="featured_image_replace_dialog_description">You already have a featured image set. Do you want to replace it with the new image?</string>
-    <string name="featured_image_replace_dialog_confirm">Replace featured image</string>
-    <string name="featured_image_replace_dialog_cancel">Keep current</string>
-    <string name="featured_image_confirmation_notice">Set as featured image</string>
-    <string name="featured_image_removed_notice">Removed as featured image</string>
-    <string name="notification_channel_general_title" tools:ignore="UnusedResources">General</string>
-    <string name="notification_channel_important_title" tools:ignore="UnusedResources">Important</string>
-    <string name="notification_channel_transient_title" tools:ignore="UnusedResources">Transient</string>
-    <string name="notification_channel_reminder_title" tools:ignore="UnusedResources">Reminder</string>
-    <string name="notification_channel_important_id" translatable="false" tools:ignore="UnusedResources">wpandroid_notification_important_channel_id</string>
-    <string name="notification_channel_reminder_id" translatable="false" tools:ignore="UnusedResources">wpandroid_notification_reminder_channel_id</string>
+
+    <!-- WordPressEditor module -->
+    <string name="post_content" tools:ignore="UnusedResources">Content</string>
```

Comment:
This one and all the ones below are because we fixed the linter error by adding `tools:ignore="UnusedResources"` in the original `WordPress/src/main/res/values/strings.xml`, so since that `strings.xml` changed, we is why that frozen `fastlane/resources/values/strings.xml` got then updated accordingly to apply the same changes made during `release/19.6` to the frozen ones.

That should not impact the GlotPress import, since that change in the frozen strings is only about moving some nodes around, and adding the `tools:ignore` attribute, but didn't change their `name` nor value/text; so I don't thing GlotPress will see when re-importing the file next time around